cypress/integration/special-cases.js

```diff
@@ -27,4 +27,16 @@ describe('special cases tests', () => {
     cy.clickMenuButton()
     articleMenuPage.getMenuOption('Language').should('not.exist')
   })
+
+  it('gallery opens from a non-english article', () => {
+    cy.navigateToPageWithoutOnboarding('article/pl/Tupolew_Tu-154')
+    articlePage.title().should('have.text', 'Tu-154')
+    for (let index = 0; index < 15; index++) {
```

Comment:
@hueitan I haven't found a way to make this test robust.
I can't do conditional testing with [cypress](https://docs.cypress.io/guides/core-concepts/conditional-testing.html#Definition), I wanted to do something like:
```
if(element is on screen)
  enter()
else
  downArrow()
```
I tried using document.querySelector but doesn't work as well.
also, since the page flipping on electron/chrome is not working properly I don't know for sure the number of page flips needed.
I tried with `index < 15` and it works on local but not on the CI.

Reply:
currently navigating doesn't work as expected, it only works for firefox

I don't have a quick fix for now, I suggest you put a comment on this skipped test so we can come back later when having the fix.